content/en/blog/_posts/2017-01-00-Running-Mongodb-On-Kubernetes-With-Statefulsets.md

```diff
@@ -4,423 +4,260 @@ date: 2017-01-30
 slug: running-mongodb-on-kubernetes-with-statefulsets
 url: /blog/2017/01/Running-Mongodb-On-Kubernetes-With-Statefulsets
 ---
-_Editor's note: Today’s post is by Sandeep Dinesh, Developer Advocate, Google Cloud Platform, showing how to run a database in a container._  
+_Editor's note: Today’s post is by Sandeep Dinesh, Developer Advocate, Google Cloud Platform, showing how to run a database in a container._
 
+{{% warning %}}
+This post is several years old. The code examples need changes to work on a current Kubernetes cluster.
+{{% /warning %}}
```

Comment:
nit: this would work better as a caution block rather than a warning one.

Reply:
I'm not familiar with the syntax. Would that be this?

```
{{< caution >}}
{{< /caution >}}
```